htdocs/contact/class/contact.class.php

```diff
@@ -113,6 +114,7 @@ class Contact extends CommonObject
 		'priv' =>array('type'=>'smallint(6)', 'label'=>'ContactVisibility', 'enabled'=>1, 'visible'=>1, 'notnull'=>1, 'position'=>175),
 		'fk_stcommcontact' =>array('type'=>'integer', 'label'=>'ProspectStatus', 'enabled'=>1, 'visible'=>-1, 'notnull'=>1, 'position'=>220),
 		'fk_prospectlevel' =>array('type'=>'varchar(12)', 'label'=>'ProspectLevel', 'enabled'=>1, 'visible'=>-1, 'position'=>255),
+		'fk_parent' =>array('type'=>'integer', 'label'=>'ParentContact', 'enabled'=>1, 'visible'=>-1, 'position'=>260),
```

Comment:
I think we should have
'enabled'=>getDolGlobalInt('SOCIETE_ENABLE_PARENT_CONTACT')
so field will be visible only when option is on.

Reply:
OK it's done